src/utils/UpdateNotification.js

```diff
@@ -61,6 +71,10 @@ define(function (require, exports, module) {
     // Data about available updates in the registry
     PreferencesManager.stateManager.definePreference("extensionUpdateInfo", "Array", []);
 
+    //Preference for hosting latest installer on a local server
+    //AUTOUPDATE_UNITTESTING
+    PreferencesManager.stateManager.definePreference("autoUpdate.testUrl", "string", "");
```

Comment:
Do you want to still retain this preference?

Reply:
@nethip , Yes we need to retain this preference, as this will be very much needed for unit testing purposes.

Reply:
Please have this preference in a separate branch. We can't be shipping with test URLs preference.

Reply:
@nethip, I have removed the preference from this branch. I will probably include the changes in a separate branch then, for unit testing purposes.